src/Modules/UI/Dnn.PersonaBar.UI/admin/personaBar/scripts/main.js

```diff
@@ -428,16 +428,44 @@ require(['jquery', 'knockout', 'moment', '../util', '../sf', '../config', './../
                     }
                 }
             },
+            /**
+             * in case path is an array, it is expected to be a sorted list of dependent sources.
+             * 
+             * @param path string|array
+             */
             loadBundleScript: function (path) {
-                if (path.indexOf('cdv=') === -1) {
-                    path += (path.indexOf('?') > -1 ? '&' : '?') + 'cdv=' + config.buildNumber;
+                function formatURL(url) {
+                    if (url.indexOf('cdv=') === -1) {
+                        url += (url.indexOf('?') > -1 ? '&' : '?') + 'cdv=' + config.buildNumber;

```

Comment:
Instead of manipulating the query string by hand please follow [this](https://github.com/dnnsoftware/Dnn.AdminExperience.Extensions/pull/503/files#diff-cd05d121830f77744d362e3214dbdacfR61) approach. It uses an external library and the util is [here](https://github.com/dnnsoftware/Dnn.AdminExperience.Extensions/blob/development/src/Modules/Content/Dnn.PersonaBar.Pages/Pages.Web/src/utils.js#L193).